src/top-navigation/use-top-navigation.ts

```diff
@@ -79,36 +79,42 @@ export function useTopNavigation({ identity, search, utilities }: UseTopNavigati
 
     // Get available width from the visible top navigation.
     const availableWidth = getContentBoxWidth(mainRef.current.querySelector(`.${styles['padding-box']}`)!);
+    /* istanbul ignore else  */
     if (availableWidth === 0) {
       // Likely in an SSR or Jest situation.
       setResponsiveState(responsiveStates[0]);
-      return;
+    } else {
+      const titleWidth = virtualRef.current.querySelector(`.${styles.title}`)?.getBoundingClientRect().width ?? 0;
+      const logoWidth = virtualRef.current.querySelector(`.${styles.logo}`)?.getBoundingClientRect().width ?? 0;
+
+      // the identity div has a flex-grow: 1 css, this makes calculating its content width requiring to calcualte both
```

Comment:
This comment is outdated now because the identity div does not have `flex-grow: 1` anymore, right?

Reply:
removed